source/core/ut_annotations.pkb

```diff
@@ -30,7 +30,7 @@ create or replace package body ut_annotations as
   c_rgexp_identifier            constant varchar2(50) := '[a-z][a-z0-9#_$]*';
   c_annotation_block_pattern    constant varchar2(200) := '(({COMMENT#.+}'||chr(10)||')+)( |'||chr(09)||')*(procedure|function)\s+(' ||
                                                            c_rgexp_identifier || ')';
-  c_annotation_pattern          constant varchar2(50) := gc_annotation_qualifier || c_rgexp_identifier || '(\(.*?\)$)?';
+  c_annotation_pattern          constant varchar2(50) := gc_annotation_qualifier || c_rgexp_identifier || '[ '||chr(9)||']*(\(.*?\)$)?';
```

Comment:
Shouldn't we use `\s*` rather than `chr(9)`?
We should also enhance the pattern to allow \s at the end I think.
So how about:
```sql
gc_annotation_qualifier || c_rgexp_identifier || '\s*(\(.*?\)\s*$)?';
```